delira/training/predictor.py

```diff
@@ -27,7 +28,9 @@ class Predictor(object):
     def __init__(
```

Comment:
I would not like to add this to a subclass, since we would have to re-derive all trainers if we intend to use TTA during training. This is why I originally came up with the idea of the decorator.